app/grandchallenge/reader_studies/forms.py

```diff
@@ -741,8 +743,55 @@ def create_ground_truth(self):
         Answer.objects.filter(
             question__reader_study=self._reader_study
         ).update(score=None)
-        bulk_assign_scores_for_reader_study.apply_async(
-            kwargs={"reader_study_pk": self._reader_study.pk}
+
+        on_commit(
+            bulk_assign_scores_for_reader_study.signature(
+                kwargs={"reader_study_pk": self._reader_study.pk}
+            ).apply_async
+        )
+
+
+class AnswersFromGroundTruthForm(SaveFormInitMixin, Form):
+    user = ModelChoiceField(
+        queryset=get_user_model().objects.none(),
+        required=True,
+        help_text=format_html(
+            "Select a user to whom the answers will assigned. "
```

Comment:
This is having one user assign answers for another. I think we want to avoid that for traceability reasons, especially thinking about the requirements that just came in and the problems that it caused with submitting on behalf of someone else. Maybe we should limit it to only being able do this for the current user?

Reply:
I think that is a very reasonable limitation. I've opted for an open-user selection initially as I could vaguely imagine someone asking to double check the answer, generally though: I suspect that it's always the editor re-assigning it to themselves.